style.css

```diff
@@ -0,0 +1,104 @@
+@import url('https://fonts.googleapis.com/css2?family=Arvo&family=Gilda+Display&family=Libre+Baskerville&family=Noto+Sans&family=Open+Sans&family=Playfair+Display:wght@800&family=Poppins:wght@700&display=swap');
+
+* {
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+body {
+  min-height: 100vh;
+  background-image: url('./assets/header-shapes-mobile.png');
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure you stick to the designs much as you can. The colors are supposed to match with the original design. kindly fix that.
`Hint:`
![image](https://user-images.githubusercontent.com/97100091/212471150-0c15b69d-5c48-4875-9a26-bf42b1cd973f.png)

You design 
![image](https://user-images.githubusercontent.com/97100091/212471014-4b3d8bb5-37e6-4214-822c-b73634f7a4cc.png)
Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/212471088-fc24da4e-fccd-43b6-9718-9388fd17f49d.png)